web/src/components/ChatController.tsx

```diff
@@ -14,48 +19,56 @@ type ChatControllerProps = {
 }
 
 const LOCAL_STORAGE_ITEM_CHAT_MESSAGES = 'Chat-Device-Id'
+const POLLING_INTERVAL = 16000
+
 const ChatController = ({ city, language }: ChatControllerProps): ReactElement => {
```

Comment:
I merged ChatInitializedView into this component for easier handling of device id, sending of messages and updating the messages.